### PR TITLE
add ggit to project schema transformer

### PR DESCRIPTION
**Pull request by jaycenhorton**

### CODE REVIEW

File: packages/project/src/utils/__tests__/fixtures.ts [hunk 1]
```diff
@@ -0,0 +1,3659 @@
+import type { Input } from '@nori-dot-com/ggit';
+
+import type { V2Data, Project } from '../../index';
+
+// todo BUG likely impacts quantification: multi tillage dates getting applied to same crop. see: 2018 no till planting for one crops and should have been for each crop
+// todo ^ check for multi tillage events on same crop
+// todo why does 4/25/2020 have a harvest date and no yield? (get back info from rebekah, adjust the below in any place yield is set to 0 in the sheet fixture)
+// todo in some years (i.e. 2019 and more) crop#2, is fertilizer applied to the correct crop? looks like it should go to the first harvested crop instead of cover crop
+
+export const v3Data: Project = {
```
Comment (jaycenhorton) [inline packages/project/src/utils/__tests__/fixtures.ts:10]: .

File: packages/project/src/utils/convertFromGgitToProject.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  ggitInputData,
+}: {
+  ggitInputData: Input.InputData;
+}): Project => {
+  const geometryData = extractGeometryData({
+    croplands: ggitInputData.Cropland,
+  });
+  const projectData: Project = { version: '0.3.0', fields: [] };
+  return projectData;
+};
```
Comment (jaycenhorton) [inline packages/project/src/utils/convertFromGgitToProject.ts:39]: .

File: packages/project/src/utils/__tests__/convertFromGgitToProject.test.ts [hunk 3]
```diff
@@ -0,0 +1,763 @@
...
+
+describe('translateCropEvent', () => {
+  it('will test the function', () => {
+    expect(translateCropEvent(null)).toStrictEqual<
+      ReturnType<typeof translateCropEvent>
+    >(null);
+  });
+});
+
+describe('extractCrops', () => {
```
Comment (jaycenhorton) [inline packages/project/src/utils/__tests__/convertFromGgitToProject.test.ts:733]: todo

**Review by scottynomad — APPROVED** [review]
Looking good!

**Review by rlfarman — APPROVED** [review]

**Pull request merged**